modules/dr-restoring-cluster-state.adoc

```diff
@@ -23,8 +23,19 @@ You should run the restore script on all of your master hosts within a short per
 +
 This procedure assumes that you have copied a snapshot file called `snapshot.db` to the `/home/core/` directory of your master host.
 
+.. If the etcd snapshot was encrypted, copy the encryption keys to the master host.
++
+This procedure assumes that you have copied the `encryption-config` file to the `/home/core/` directory of your master host.
+
 .. Access the master host.
 
+.. If the etcd snapshot was encrypted, restore the `encryption-config` file to the proper location.
```

Comment:
s/proper/appropriate

> ! proper adj
Do not use to mean “correct,” “applicable,” or “appropriate.”

Is there more nuance than just "put it in the most recent revision of the `kube-apiserver-pod-<revision>` directory?" If not, consider ~to the proper location~ instead.